Fix/lti consumer (#1374)
openfun/richie: CHANGELOG.md

- Send username to LTI Consumer if user is logged in
- Make related courses optional for program page

### Fixed

> - Use a unique name to LTIConsumerPlugin iframes
>   to avoid a bug when several plugins was on the same page

Reviewer: ```suggestion
- Use a unique name to LTIConsumerPlugin iframes
  to allow several plugins on the same page
```

> - Change logic to retrieve LTIConsumerPlugin component parent DOM node to avoid
>   an error that occured on iframeResizer instantiation when several plugins was
>   on the same page

Reviewer: Maybe too long and too vague?
```suggestion
- Fix iframeResizer on LTIConsumer plugin when several plugins are on the same page
```
